small/slab_cache.c

```diff
@@ -316,6 +316,8 @@ slab_put_large(struct slab_cache *cache, struct slab *slab)
 struct slab *
 slab_get(struct slab_cache *cache, size_t size)
 {
+	if (size + slab_sizeof() > UINT32_MAX)
+		return NULL;
```

Comment:
I'm not sure we should fail in this case. Maybe, fallback on malloc? This needs to be discussed with @alyapunov.

Reply:
BTW `slab_get_large`, which is called by `slab_get`, already falls back on malloc. Looks like this patch is wrong and we should instead fix `slab_order` to return `cache->order_max + 1` in this case.

Reply:
Yes you right, remove assertion, now we return cache->order_max + 1 and fall to `slab_get_large`

Reply:
Let's just change unsigned to uint64_t and __builtin_clz to __builtin_clzll.